common/flatpak-dir.c

```diff
@@ -8886,6 +8914,7 @@ flatpak_dir_update_remote_configuration (FlatpakDir   *self,
     return flatpak_dir_update_remote_configuration_for_repo_metadata (self, remote, summary, FALSE, NULL, cancellable, error);
 }
 
+/* TODO: Move these sizes out to per-ref or per-commit metadata */
```

Comment:
> This comment scares me a bit. We don't want to be in the situation where we have gnome-software blocking on startup until it has downloaded all the commit objects in all remotes before it starts showing anything, that will be very slow.

The comment’s not very well worded, and it was more of a note to myself than to others. It’s to remind me to look into moving the entries from `xa.cache` into the additional metadata for each ref in the `summary` file (i.e. the `a{sv}` in the 0th child of `OSTREE_SUMMARY_GVARIANT_FORMAT`) so that the `xa.cache` property itself is split up and eliminated.

The bit about ‘per-commit metadata’ is just wrong.